src/v2/Components/NavBar/Menus/__tests__/NotificationsMenu.jest.tsx

```diff
@@ -9,13 +9,18 @@ describe("NotificationsMenu", () => {
   const trackEvent = jest.fn()
 
   beforeEach(() => {
+    jest.spyOn(console, "error").mockImplementation(() => {})
```

Comment:
What's up with these around now, is there some path that logs an error?

Reply:
So, it's the [unknown props](https://reactjs.org/warnings/unknown-prop.html) thing. Basically; React logs an error, but it also essentially doesn't matter. And there's not a fantastic way to fix this until we upgrade styled-components. We can use CleanTag but there's been some issues with typing — particularly around the use of `as`.

* I do not understand why React logs an error for these instead of a `console.warn`, it literally starts the error with `Warning: `
* There is actual no discussion of why this is even bad in the first place. It has no actual implications as far as I can tell aside from just being ugly, sending more HTML to the client.

We need to either:
* Upgrade to styled-components v5 and filter the props (ideal) — almost there: current blockers I'm running into right now are issues with the `primitives.View` pattern in palette interaction with Jest's generics (I *think*).
* Implement a better version of CleanTag
* Disable Jest's treating of `console.error` as actual errors